### PR TITLE
feat: fuelup completions

### PR DESCRIPTION
**Pull request by 0xYYY**
Closes https://github.com/FuelLabs/fuelup/issues/291.

### CODE REVIEW

File: src/fuelup_cli.rs [hunk 1]
```diff
@@ -22,6 +23,8 @@ struct Cli {
 enum Commands {
     /// Check for updates to Fuel toolchains and fuelup
     Check(CheckCommand),
+    /// Generate shell competions
```
Comment (bingcicle) [inline src/fuelup_cli.rs:26]:
```suggestion
    /// Generate shell completions
```

**Review by bingcicle — CHANGES_REQUESTED** [review]
Thanks for your contribution! Looks awesome - just a few nits :)

File: src/fuelup_cli.rs (hunk 1 above)
Comment (0xYYY) [reply src/fuelup_cli.rs:26]: thanks for catching this :)

File: fuelup-init.sh [hunk 2]
```diff
@@ -137,7 +137,7 @@ main() {
     ignore rmdir "$_dir"
 
     printf '\n'
-    printf '%s\n' "fuelup ${_fuelup_version} has been installed in $FUELUP_DIR/bin. To fetch the latest toolchain containing the forc and fuel-core binaries, run 'fuelup toolchain install latest'." 1>&2
+    printf '%s\n' "fuelup ${_fuelup_version} has been installed in $FUELUP_DIR/bin. To fetch the latest toolchain containing the forc and fuel-core binaries, run 'fuelup toolchain install latest'. To generate completions for your shell, run 'fuelup completions --shell=SHELL'." 1>&2
```
[the remark this answers is not in the document]
Comment (0xYYY) [reply fuelup-init.sh:140]: didn't realise that the book is also in this repo
will add some instructions

**Comment by 0xYYY** [comment]
hi @bingcicle, thanks for your suggestions, i've updated accordingly

(book content for `fuelup completions` is adapted from https://fuellabs.github.io/sway/master/forc/commands/forc_completions.html)

**Review by bingcicle — APPROVED** [review]
LGTM, thank you for your contribution!

**Pull request merged**